tests/streams.test.cpp

```diff
@@ -97,8 +96,7 @@ void stream_terminated_test()
 //                              |  parse<T> Stream  |
 //
 // =============================================================================
-void parse_stream_test()
-{
+boost::ut::suite<"streams"> parse_stream_test = [] {
```

Comment:
Can you name these after the type of test. Like this one should be parse stream test.